### ISSUE TITLE
1.14 update

### ISSUE THREAD

**Issue by jonttm** [graywolf336/BestViewDistance#10]
Is there any possibility of 1.14 support? I know the original developer has kind of stopped working on the project since January and you continued it with bug fixes. Some of my players have fairly bad internet and they have trouble connecting because of loading terrain (even at 10 chunks). It would be great if we could get 1.14+ support in the future.

**Comment by graywolf336** [comment]
Have you tried it yet on 1.14? I'd be curious if the current builds work or not

**Comment by jonttm** [comment]
Yes, I tried it a few weeks ago actually. It spams several errors in console and the view distance is stuck at what server.prop has while none of the commands work.

**Comment by jonttm** [comment]
I will test again tomorrow and put some of the console output here.

**Comment by graywolf336** [comment]
Yeah, if you can show the errors that would be extremely helpful. I haven't had time to try out 1.14 and don't care to until they fix several of the major issues that I've seen

**Comment by jonttm** [comment]
Here's a full console output:
https://pastebin.com/1bRZfBDH

I created a fresh server and then dropped the plugin jar in. First it disabled because of ProtocolLib not being present (I swore this was only 1.12 and below?). I added ProtocolLib and it complained about spigot.yml view-distance being set to 0 (note that the view-distance setting in spigot.yml is no longer present in 1.14). I joined the game and it starts sending errors in console every 20-30 seconds (presumably during calculations). My view distance doesn't change in-game. I tried a few commands like /vdist and /view and they returned "an internal error occurred" in chat and the errors you see in console. Let me know if you need more information.

**Comment by graywolf336** [comment]
They haven't updated the API for setting custom view distance:

`org.apache.commons.lang.NotImplementedException: Per-Player View Distance APIs need further understanding to properly implement`

So, there is currently no way to do that  in 1.14.

**Comment by jonttm** [comment]
Thanks for the info!

**Comment by graywolf336** [comment]
Going to reopen this for others to see :)

**Comment by jonttm** [comment]
Good idea
